Sources/DatadogObjc/DDURLSessionDelegate+objc.swift

```diff
@@ -0,0 +1,11 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import Datadog
+
+@objc
+public class DDNSURLSessionDelegate: DDURLSessionDelegate {}
```

Comment:
re-posting previous comment: can't we remove this file?

Reply:
As explained (https://github.com/DataDog/dd-sdk-ios/pull/262#discussion_r500047607), nope - the delegate definition must be exported from `DatadogObjc.framework`. We do the same for all other things in Objc SDK.